src/macros.rs

```diff
@@ -327,3 +327,148 @@ macro_rules! template {
     };
     () => {}
 }
+
+/// Utility macro for conditionally joining labels, such as `String` or `&str`,
+/// or anything that implements the `RenderOnce` trait can be used.
+///
+/// This macro is an alias of: `labels_sep_by!(" "; maybe_label,...)`
+///
+/// Usage: `labels!(maybe_label,...)`
+///
+/// * `maybe_label` -- Either `label_expression`, or `label_expression => cond_test`.
+///
+/// * `label_expression` -- An expression that returns a label that implements
+///   the `RenderOnce` trait (e.g. `String` or `&str`).
+///
+/// * `label_expression => cond_test` -- Conditionally include `label_expression` whenever `cond_test` is `true`.
+///   `cond_test` is an expression that returns either `true` or `false`.
+///
+/// This useful in generating class attribute as follows:
+///
+/// ```norun
```

Comment:
That's because I've been lazy :sleeping:. I just pushed a commit that removes the norun directives (so you can see how to do this). (...I also ended up having to fix many of the examples which is why norun should be avoided...).